html/includes/table/eventlog.inc.php

```diff
@@ -60,12 +60,16 @@
         $type = $eventlog['type'];
     }
     $severity_colour = $eventlog['severity'];
+    if ($eventlog['username'] === 0) {
```

Comment:
Are you sure the identical comparison (===) is the correct one here?
Aren't all db results strings?